README/readme.md

```diff
@@ -44,68 +48,70 @@ The basic Open ID Connect set is:
 
 * User info request
 
-To these one can add services like session management and token introspection. The only thing we can be sure of is that 
-this is not the final set of services, there will be more. And there will be variants of the standard ones. Like when you
-want to add multi lateral federation support to provider information discovery and client registration.
+To these one can add services like session management and token introspection. The
+only thing we can be sure of is that this is not the final set of services, there
+will be more. And there will be variants of the standard ones. Like when you want 
+to add multi lateral federation support to provider information discovery and
+client registration.
 
-Over all it seemed like a good idea to write a piece of code that implements all the functionality that is needed to support
-any of this services and any future services that follows the same pattern.
+Over all it seemed like a good idea to write a piece of code that implements all 
+the functionality that is needed to support any of this services and any future 
+services that follows the same pattern.
 
 
 ### The request pipeline
-Below follows a desciption of the parts of the request pipeline in the order they are called.
+Below follows a description of the parts of the request pipeline in the order they
+are called.
 
 The overall call sequence looks like this:
-
     do_request_init
```

Comment:
You need to keep a newline before here otherwise it's not properly formatted (see https://github.com/GJWT/nodeOIDCCli/blob/c425742916b631d95a43d47f68a69466cf3294f1/README/readme.md)